src/config/redis-info.ts

```diff
@@ -6,7 +6,14 @@ export const getRedisInfo = (connectionName: string): IORedis.RedisOptions => ({
 	port: Number(envVars.REDISX_CACHE_PORT) || 6379,
 	host: envVars.REDISX_CACHE_HOST || "127.0.0.1",
 	db: 0,
+	reconnectOnError(err) {
+		const targetError = "READONLY";
```

Comment:
considered making `const targetErrors = ["READONLY", "BLAH BLAH"];` work, but the reconnection logic that is handled by `ioredis` was working for the examples that I could find.... If we run into problems in the future will be pretty easy to slap it in.